src/createServer.js

```diff
@@ -1,11 +1,23 @@
+/* eslint-disable padding-line-between-statements */
```

Comment:
Consider removing the eslint-disable line and add padding-line-between-statements as needed for better readability.